pkgs/applications/virtualization/libgovirt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchurl
+, glib
+, librest
+, libsoup
+, pkg-config
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "libgovirt";
+  version = "0.3.8";
+
+  src = fetchurl {
+    url = "https://download.gnome.org/sources/libgovirt/0.3/${pname}-${version}.tar.xz";
+    sha256 = "sha256-HckYYikXa9+p8l/Y+oLAoFi2pgwcyAfHUH7IqTwPHfg=";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  propagatedBuildInputs = [
+    librest
+    libsoup
+  ];
+}
```

Comment:
Could you please add a `meta` block?

Reply:
Ah yes of course, apologies. Added and rebased. Added myself as a maintainer since I’m adding the package, let me know if that doesn’t make sense.

Reply:
That's the ideal case, yes..